bitnami/cassandra/values.yaml

```diff
@@ -17,7 +17,7 @@ image:
   ## Bitnami Cassandra image tag
   ## ref: https://github.com/bitnami/bitnami-docker-cassandra#supported-tags-and-respective-dockerfile-links
   ##
-  tag: 3.11.10-debian-10-r23
+  tag: 3.11.10-debian-10-r6
```

Comment:
Could you change the tag to `3.11.10-debian-10-r23`?

Reply:
Done